app/services/locations/finder.rb

```diff
@@ -67,5 +68,13 @@ def apply_young_offender_institution_filters(scope)
       scope = scope.where(young_offender_institution: false) if filter_params[:young_offender_institution].to_s == 'false'
       scope
     end
+
+    def apply_created_at_filters(scope)
+      return scope if filter_params[:created_at].blank?
+
+      date = Date.strptime(filter_params[:created_at])
```

Comment:
elsewhere we have been explicit about the date format, e.g. `Date.strptime(value, '%Y-%m-%d')` - I think it would be good to do so here to avoid any future misunderstanding